src/types.ts

```diff
@@ -23,6 +23,23 @@ export interface Environment {
 }
 
 export interface Preset {
+  meta?: {
```

Comment:
do we want name/version (at least) as read-only?

Reply:
actually whole preset shoudln't be mutated.. marked new keys as readonly.